Libraries/LibWeb/Selection/Selection.cpp

```diff
@@ -8,6 +8,7 @@
 #include <LibWeb/Bindings/Intrinsics.h>
```

Comment:
Commit message:
> LibWeb: Selection::getRangeAt handle focus and anchor

Make that a sentence, e.g.
> LibWeb: Update Selection::getRangeAt() spec to handle focus and anchor

This is also not a complete sentence and lacks punctuation:
> Following a spec change, need to check for focus and anchor in range

Reply:
Yeah, that's a leftover from the previous committer :)
Fixed

Reply:
Oh that's right, in that case you'd mention both authors with `Co-authored-by` tags in your commit message :)

Reply:
There's nothing left from the previous PR in this PR
Not a line of code :)

Reply:
It includes a line proposed by @kalenikaliaksandr at https://github.com/LadybirdBrowser/ladybird/pull/1767#discussion_r1798470277
I'm not sure if I should mention them as coauthor

Reply:
Well who cares anyway let's list them all and rock